packages/backend/src/plugins/search.ts

```diff
@@ -35,11 +34,7 @@ export default async function createPlugin({
     collator: new DefaultCatalogCollator(discovery),
   });
 
-  // TODO: Move refresh loop logic into the builder.
-  const timerId = setInterval(() => {
-    indexBuilder.build();
-  }, 60000);
-  useHotCleanup(module, () => clearInterval(timerId));
+  indexBuilder.build();
```

Comment:
Should this method still be called `build` when it actually starts a background process that later triggers builds?

Reply:
👍, I think we can find a better name for it now!